### PR TITLE
Target and release column in images table

### PR DESCRIPTION
**Pull request by mgold1234**
# Description

this commit add 2 new columns at images table and change the order of the column like requested in the mockup image that appear in the ticket

Fixes # (issue) -related to that ticket  -https://issues.redhat.com/browse/THEEDGE-3311

## Type of change

What is it?

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Documentation update
- [ ] Tests update

# Checklist:

- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [x] I run `npm run lint:js:fix` to check that my code is properly formatted

### CODE REVIEW

**Review by matoval — CHANGES_REQUESTED** [review]
I left an inline comment about the release cell. 
Another issue is sorting isn't working.

File: src/Routes/ImageManager/ImageSetsTable.js [hunk 1]
```diff
@@ -87,13 +100,13 @@ const createRows = (data) => {
...
       },
       image_set?.Version,
-      {
-        title: image_set?.UpdatedAt ? (
-          <DateFormat date={image_set?.UpdatedAt} />
-        ) : (
-          'Unknown'
-        ),
-      },
+      image_set?.Distribution,
```
[the remark this answers is not in the document]
Comment (mgold1234) [reply src/Routes/ImageManager/ImageSetsTable.js]: done

[the remark this answers is not in the document]
Comment (mgold1234) [reply src/Routes/ImageManager/ImageSetsTable.js]: remove sort from Target and Release column

File: src/Routes/ImageManager/ImageSetsTable.js [hunk 2]
```diff
@@ -91,12 +106,14 @@ const createRows = (data, baseURL, history, navigate) => {
...
-          'Unknown'
-        ),
+        title: distributionMapper[image_set?.Distribution],
       },
+      image_set?.OutputTypes[0] == 'rhel-edge-installer' &&
+      image_set?.OutputTypes[1] == 'rhel-edge-commit'
+        ? 'Bare Metal Installer'
+        : image_set?.OutputTypes[0] == 'rhel-edge-commit'
+        ? 'Update Only'
+        : null,
```
Comment (ldjebran) [inline src/Routes/ImageManager/ImageSetsTable.js:116]: in my local database the OutputTypes looks like {"rhel-edge-commit","rhel-edge-installer"} so the commit is first,
maybe we should simply check if image_set?.OutputTypes.length == 2 ? 'Bare Metal Installer' :  'Update Only'
what do you think ?

Comment (mgold1234) [reply src/Routes/ImageManager/ImageSetsTable.js:116]: good idea

Comment (ldjebran) [reply src/Routes/ImageManager/ImageSetsTable.js:116]: or if you need to be more explicit: 

image_set?.OutputTypes.includes('rhel-edge-installer') it's 'Bare Metal Installer' else if image_set?.OutputTypes.includes('rhel-edge-commit') then 
'Update Only' else 'unknown'

**Review by ldjebran — APPROVED** [review]
lgtm

**Review by matoval — APPROVED** [review]

**Pull request merged**